ppr-ui/src/components/common/QsAccessBtn.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="qs-access-btn">
+  <div id="qs-access-btn" role="button" aria-label="qualified-supplier-access-request-btn">
```

Comment:
An example of an AA issue, picked up and subsequently fixed by our jest. Very minimal stuff and this is the vast majority of the type of issues.